### ISSUE TITLE
Changelog displayed by "Plugin Changelog" always show "Chore"

### ISSUE THREAD

**Issue by stecydube** [SilentVoid13/Templater#509]
I use the "Plugin Changelog" to see the changelog but under every version it is always, for example:

    chore(release): 1.9.9

I guess the plugin is taking the text under the "releases" thread and not the changelog.md file.

Maybe having the text in the changelog for a particular release also be added to the description of the release? 😉

For an example of how to do it: 
https://github.com/valentine195/obsidian-admonition/commit/71bb9a7d5b805133fe648fd95f880515d45cff61
and
https://github.com/valentine195/obsidian-admonition/commit/e9ba91f727a4c61ab9249f668c18760679616b2c

**Comment by shabegom** [comment]
Good point. In the interim you can see changes in the Changelog.md file
